ai-compile/vscode-extension/src/codeActionProvider.ts

```diff
@@ -37,16 +38,17 @@ export class EduCodeActionProvider implements vscode.CodeActionProvider {
   
     let lineText = document.lineAt(fix.lineNo).text;
     lineText = lineText.slice(edit.start);
-    const startCharCode = lineText.charCodeAt(0);
+    const lineStartCharCode = lineText.charCodeAt(0);
     
-    // underline next word (in case of a starting WS character - typical of indentation fixes)
-    if(startCharCode === " ".charCodeAt(0)) {
+    // capture range of next word (in case of a starting whitespace (WS) character - typical of indentation fixes)
+    if(lineStartCharCode === " ".charCodeAt(0)) {
       
-      const alNumRegExp = /[a-z0-9_]+/ig;
```

Comment:
Use some built-in library for this and do not reimplement checking if something is alphanumeric.

Reply:
Could you please point me to any built-in library for achieving this? I searched for it online and couldn't really find one, that's probably why I couldn't include it in the first place.
There are 3rd party libraries however, which isn't much different from the above implementation, given the trivial nature of the implementation.